src/controllers/organisations.ts

```diff
@@ -227,3 +227,26 @@ export const deleteOrganisation = asyncHandler(async (req, res, next) => {
         data: rows,
     });
 });
+
+/**
+ * @desc    Soft delete single organisation
+ * @route   PUT /api/organisations/:id
+ * @access  Admin/Owner
+ */
+export const deactivateOrganisation = asyncHandler(async (req, res, next) => {
+    const { id } = req.params;
+    const userId: string = req.user.user_id;
```

Comment:
Can be written as
```suggestion
    const { user_id: userId }: string = req.user;
```

Reply:
can't change to const { user_id: userId }: string = req.user; because user_id does not exist on type string.

Reply:
I see, my bad, it is a typescript syntax issue. How about this?

```suggestion
    const { user_id: userId }: { user_id: string, [key: string]: string } = req.user;
```

This will assign the variable `userId` to `req.user.user_id` and tell TS to infer `userId` as type `string`